netsuite/client.py

```diff
@@ -145,6 +147,7 @@ def __init__(
         cache: zeep.cache.Base = None,
         session: requests.Session = None,
         sandbox: bool = None,
+        args: argparse.Namespace = argparse.Namespace(timeout=60)
```

Comment:
Thanks for your submission @onnimonni. Tying `argparse.Namespace` to the `NetSuite` client is a bit limiting. Perhaps look into adding a `default_rest_api_timeout` keyword argument instead?